src/platform/stm32f1/usb_descriptors.c

```diff
@@ -0,0 +1,185 @@
+	(u8)((USB_VID >> 8) & 0x00ff),
+	(u8)(USB_PID & 0x00ff),			/* PRODUCT ID */
+	(u8)((USB_PID >> 8) & 0x00ff),
+	0x00, 0x01,			/* DEVICE RELEASE */
+	0x01,				/* MANUFACTURER STRING INDEX */
+	0x02,				/* PRODUCT STRING INDEX */
+	0x00,				/* SERIAL STRING INDEX (None) */
+	0x01,				/* NO CONFIGURATIONS */
+	/* clang-format on */
+};
+
+/* HID Mouse report descriptor */
+const u8 desc_hid_mouse_report[] = {
+	/* clang-format off */
+	0x05, 0x01,	/* USAGE_PAGE (Generic Desktop) */
+	0x09, 0x02,	/* USAGE (Mouse) */
+	0xa1, 0x01,	/* COLLECTION (Application) */
+	0x09, 0x01,		/* USAGE (Pointer) */
+	0xa1, 0x00,		/* COLLECTION (Physical) */
+	0x85, 0x01,			/* REPORT_ID (0x01) */
+	0x05, 0x01,			/* USAGE_PAGE (Generic Desktop) */
+	0x09, 0x30,			/* USAGE (X) */
+	0x09, 0x31,			/* USAGE (Y) */
+	0x09, 0x38,			/* USAGE (WHEEL) */
+	0x15, 0x81,			/* LOGICAL_MINIMUM (-127) */
+	0x25, 0x7f,			/* LOGICAL_MAXIMUM (127) */
+	0x75, 0x08,			/* REPORT_SIZE (8) */
+	0x95, 0x03,			/* REPORT_COUNT (3) */
+	0x81, 0x06,			/* INPUT (Data,Var,Rel) */
+	0x05, 0x09,			/* USAGE_PAGE (Button) */
+	0x19, 0x01,			/* USAGE_MINIMUM (Button 1) */
+	0x29, 0x03,			/* USAGE_MAXIMUM (Button 3) */
+	0x15, 0x00,			/* LOGICAL_MINIMUM (0) */
+	0x25, 0x01,			/* LOGICAL_MAXIMUM (1) */
+	0x95, 0x03,			/* REPORT_COUNT (3) */
+	0x75, 0x01,			/* REPORT_SIZE (1) */
+	0x81, 0x02,			/* INPUT (Data,Var,Abs) */
+	0x95, 0x01,			/* REPORT_COUNT (1) */
+	0x75, 0x05,			/* REPORT_SIZE (5) */
+	0x81, 0x01,			/* INPUT (Cnst,Var,Abs) */
+	0xc0,			/* END_COLLECTION */
+	0xc0,		/* END_COLLECTION */
+	/* clang-format on */
+};
+
+/* HID Report Descriptor */
+u8 desc_hid_report[sizeof(oi_rdesc) + sizeof(desc_hid_mouse_report)];
+
+/* Configuration Descriptor */
+const u8 desc_configuration[] = {
+	/* clang-format off */
+    /* configuration */
+	0x09,           /* LENGTH */
+	0x02,	        /* DESCRIPTOR TYPE (Configuration) */
+	0x22, 0x00,	    /* TOTAL LENGTH [Configuration(9) + Interface(9) + HID(9) + Endpoint(7) = 34] */
+	0x01,		    /* NUM INTERFACES (1) */
+    0x01,           /* CONFIG NUMBER (1) */
+    0x00,           /* STRING INDEX (null) */
+    0xA0,           /* ATTRIBUTES (Bus powered, Remote wakeup) */
+    0x32,			/* MAX POWER (100mA) */
+    /* Interface */
+    0x09,	        /* LENGTH */
+	0x04,	        /* DESCRIPTOR TYPE (Interface) */
+	0x00,	        /* INTERFACE NO (0) */
+	0x00,		    /* ALTERNATE SETTING (none) */
+    0x01,           /* NUM ENDPOINTS (1) */
+    0x03,           /* INTERFACE CLASS (HID) */
+    0x00,           /* INTERFACE SUBCLASS (None) */
+    0x00,           /* INTERFACE PROTOCOL (None) */
+    0x00,           /* INTERFACE STRING (Null) */
+    /* HID */
+    0x09,	        /* LENGTH */
+	0x21,	        /* DESCRIPTOR TYPE (hid) */
+	0x11, 0x01,	    /* HID VERSION (0x0111) */
+	0x00,		    /* COUNTRY CODE (None) */
+    0x01,           /* NO DESCRIPTORS (1) */
+    0x22,           /* DESCRIPTOR TYPE (Report) */
+    sizeof(oi_rdesc) + sizeof(desc_hid_mouse_report),
```

Comment:
Should be `sizeof(desc_hid_report)` to avoid mismatching if we change something.